src/Raven.Server/Json/BlittableJsonTextWriterExtensions.cs

```diff
@@ -689,11 +689,11 @@ public static async Task<(long NumberOfResults, long TotalDocumentsSizeInBytes)>
                     (w, c, spatialShape) => w.WriteSpatialShapeResult(c, spatialShape));
             }
 
-            if (result.RaftCommandIndex.HasValue)
+            if (result.IndexDefinitionRaftIndex.HasValue)
             {
                 writer.WriteComma();
-                writer.WritePropertyName(nameof(result.RaftCommandIndex));
-                writer.WriteInteger(result.RaftCommandIndex.Value);
+                writer.WritePropertyName(nameof(result.IndexDefinitionRaftIndex));
```

Comment:
Is is a breaking change?

Reply:
No, it was introduced in 6.0:

https://github.com/ravendb/ravendb/pull/14112/files#diff-34342d101561a370130f2ace8940f2a0f876375b0e9f5e693a31c98a76f7a0edR62-R63